geomstats/visualization/discrete_curves.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python
+
+"""Discrete_curve.py file details."""
+
+__authors__ = "Jax Burd & Abhijith Atreya"
+__course__ = "UCSB ECE 594N"
+__professor__ = "Nina Miolane"
+__deadline__ = "Thursday, 04/21/2022"
+
+# -------------------------------------------------------------------------------------------------------------------
+
+# IMPORTS
+import matplotlib.pyplot as plt
+import numpy as np
+from geomstats.geometry.discrete_curves import DiscreteCurves
+from geomstats.geometry.euclidean import Euclidean
+
+
+# CLASS
+class DiscreteCurveViz:
+    """Space of discrete curves sampled at points in ambient_manifold.
+
+    Each curve is represented by a 2d-array of shape `[n_sampling_points, ambient_dim]`.
+    A Batch of curves can be passed to
+    all methods either as a 3d-array if all curves have the same number of
+    sampled points, or as a list of 2d-arrays, each representing a curve.
+
+    Parameters
+    ----------
+    curve_dimension : Manifold
+        Manifold in which curves take values.
+
```

Comment:
Remove blank line between parameter description, here and everywhere in the docstrings